### PR TITLE
Add webviz-dev workflow

### PR DESCRIPTION
**Pull request by troygibb**
## Summary

Currently we have a pretty non-intuitive way to dev on Webviz, which is via `npm run docs` and navigating to `http://localhost:8080/app` to view Webviz. 

This update adds a quick hack around that, by re-tooling our Webpack config with a `WEBVIZ_DEV` flag that sets up all the relative paths and html-webpack-plugin.

Other things:
- Added a quick update to the docs as well to reflect this new workflow.
- I noticed a few `javascript heap out of memory` errors-- hence `NODE_OPTIONS='--max-old-space-size=4096'`

## Test Plan

Verified things looked good locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,24 @@ Similarly to recording videos, we have a script to measure the performance of a
...
+
+### Docs
+
+```npm run docs```
+
+### Storybook
+
+```npm run storybook```
+
+Useful for `regl-worldview` changes.
```
Comment (vidaaudrey) [inline README.md:83]: The other way to see worldview changes is to run `npm run watch` and `npm run docs` in two different tabs. Maybe we could combine and create `docs dev`?

File: README.md [hunk 2]
```diff
@@ -66,6 +66,24 @@ Similarly to recording videos, we have a script to measure the performance of a
...
+
+### Full list of commands
```
Comment (vidaaudrey) [inline README.md:85]: These are called `scripts` in the npm :) Technically we have other scripts as well, maybe change to `Other scripts`?

File: package.json [hunk 3]
```diff
@@ -133,7 +134,7 @@
     "build-storybook": "build-storybook -c stories",
     "flow": "flow",
     "flow-typed-rebuild": "rm -rf flow-typed/ && flow-typed install && flow-typed install --rootDir packages/@cruise-automation/hooks && flow-typed install --rootDir packages/@cruise-automation/button && flow-typed install --rootDir packages/@cruise-automation/tooltip && flow-typed install --rootDir packages/regl-worldview",
-    "docs": "DEV_SERVER=true webpack-dev-server",
+    "docs": "NODE_OPTIONS='--max-old-space-size=4096' DEV_SERVER=true webpack-dev-server",
```
Comment (vidaaudrey) [inline package.json:137]: Curious did you hit some performance issue in the open source version or just thought it's a good idea to add `--max-old-space-size`?

**Review by vidaaudrey — APPROVED** [review]
LGTM. A few nit comments.

File: package.json (hunk 3 above)
Comment (troygibb) [reply package.json:137]: i mention it in the pr description-- i was hitting errors without this option

File: README.md (hunk 1 above)
Comment (troygibb) [reply README.md:83]: docs-dev is good! I'll updated

File: README.md (hunk 2 above)
Comment (troygibb) [reply README.md:85]: Will do

**Pull request merged**